### DISCUSSION TITLE
为什么「中性色」和「功能色」的 Alias Token 分类逻辑不一样？是有什么考虑吗？

### DISCUSSION THREAD

**Discussion by zyf394** [ant-design/ant-design#41414] [Q&A] [answered]
中性色：按等级分类
- 一级文本色 colorText
- 二级文本色 colorTextSecondary
- 三级文本色 colorTextTertiary
- 四级文本色 colorTextQuaternary

功能色：按使用场景分类
- 主色浅色背景色 colorPrimaryBg
- 主色浅色背景悬浮态 colorPrimaryBgHover
- 主色描边色 colorPrimaryBorder
- 主色描边色悬浮态 colorPrimaryBorderHover
- 主色悬浮态 colorPrimaryHover
- 品牌主色 colorPrimary
- 主色激活态 colorPrimaryActive
- 主色文本悬浮态 colorPrimaryTextHover 
- 主色文本 colorPrimaryText
- 主色文本激活态 colorPrimaryTextActive

**Comment by arvinxx** [comment, marked as answer]
我们的基本原则是 map token 尽量语义化。功能色直接语义化没有出现什么问题。

但是在尝试将中性色语义化的过程中，发现中性色的使用场景和范围比功能色复杂很多。最后探索下来中性色中的 text 和 fill 采用了梯度。

**Reply by zyf394** [reply]
Thanks~
